### DISCUSSION TITLE
Welcome to openapi-componentizer Discussions!

### DISCUSSION THREAD

**Discussion by bavulapati** [postman-open-technologies/openapi-componentizer#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about your interest in this project.

**Comment by Polaris66** [comment]
Hi, I am Dhruv Arora, a second year B Tech CSE Student. 

This project I feel would be a great addition in my open source journey. I hope to learn more about working with the OpenAPI Specification and development CLI applications. 

For the qualification task, I was wondering if it would be possible to get an example of url and type values for testing the application.

Also, is there any way to discuss draft proposals and get ideas for improvement?

I'm really grateful for the opportunity to be able to work on the qualification task and hopefully, the project and be able to contribute to the OpenAPI spec and Postman.

**Reply by wldeh** [reply]
Hey Dhruv! It's great to hear about your interest with this project. Reaching out to your assigned mentor for input on your project proposal is definitely something you should do.
